src/bluetooth-fw/nimble/pairing_confirm.c

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
This and the other NPL files are based upon the FreeRTOS versions - how should I correctly attribute this? They are both Apache 2.0 licensed.